src/style.css

```
@@ -1 +1,86 @@
-/* add styles here */
+@font-face {
+  font-weight: 400;
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
```

Body doesn't have a paddings
```suggestion
body {
  margin: 0;
}
```